### PR TITLE
Add generic other_insert_type field in run element and sample to stor…

### LINKED ISSUE #147: Add schema for multiple insert size type
Although we expect all the read to describe and insert by pointing toward each other it is possible that some will be oriented differently, here are the three possible categories:
 1. FR: (forward reverse) the reads point toward each other
 2. RF: (reverse forward) the reads turn their back on each other.
 3. TANDEM: the read faces the same direction.
When a high enough number of read show up in category 2 or 3, the corresponding insert size is reported and will be stored.

We need to add facility to store them in the schema.

### PR DESCRIPTION
**Pull request by tcezard**
…e additional inserts metrics
fixes #147

### CODE REVIEW

File: etc/schema.yaml [hunk 1]
```diff
@@ -62,7 +62,8 @@ run_elements:
         mean_insert_size: { type: 'float' },
         std_dev_insert_size: { type: 'float' },
         median_insert_size: { type: 'integer' },
-        median_abs_dev_insert_size: { type: 'integer' }
+        median_abs_dev_insert_size: { type: 'integer' },
+        other_insert_type: { type: 'dict' }
```
Comment (mwhamgenomics) [inline etc/schema.yaml]: Is there a more descriptive name than `other_insert_type`? At the moment, we're referring to inserts that are not forward-reverse facing - something like non_facing_inserts, maybe?

Comment (tcezard) [reply etc/schema.yaml]: I guess that is debatable and I don't have very strong preferences.

File: etc/schema.yaml [hunk 2]
```diff
@@ -133,6 +135,7 @@ samples:
     std_dev_insert_size: {      type: 'float' }
     median_insert_size: {       type: 'integer' }
     median_abs_dev_insert_size: { type: 'integer' }
+    other_insert_type: {        type: 'dict', required: False }
```
Comment (mwhamgenomics) [inline etc/schema.yaml]: If we're switching to `non_fr_inserts`, we'd need to change this as well

Comment (tcezard) [reply etc/schema.yaml]: you're too quick to review.
Didn't have time to push.

**Pull request merged**